sequence_processing_pipeline/PipelineError.py

```diff
@@ -3,3 +3,31 @@ def __init__(self, message=None, mailing_list=None):
         self.message = message
         self.mailing_list = mailing_list
         super().__init__(self.message)
+
+
+class JobFailedError(PipelineError):
+    # Occurs when a successfully-submitted job as failed.
+    def __init__(self, message=None, mailing_list=None):
+        self.message = message
+        self.mailing_list = mailing_list
+        super().__init__(self.message)
+
+
+class ExecFailedError(PipelineError):
+    # Occurs when an executed command returns w/an error, which is defined as
+    # the command returning a value not zero and not an acceptable non-zero
+    # value.
+    def __init__(self, message=None, mailing_list=None):
+        self.message = message
+        self.mailing_list = mailing_list
+        super().__init__(self.message)
+
+
+class LogParsingError(PipelineError):
+    # Occurs when an executed command returns w/an error, which is defined as
+    # the command returning a value not zero and not an acceptable non-zero
+    # value. May or may not be useful.
+    def __init__(self, message=None, mailing_list=None):
+        self.message = message
+        self.mailing_list = mailing_list
```

Comment:
I guess the idea is that the errors/messages can be sent to the mailing list, right? Is this actually implemented?

Reply:
I checked metapool, qp-klp, and even the Qiita codebase and none are using the optional mailing_list parameter. I believe it's a holdover that wasn't removed before merging. I've removed them in this latest PR.